providers/catalog.go

```diff
@@ -287,4 +288,23 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Zoom configuration
+	Zoom: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.zoom.us",
+		OauthOpts: OauthOpts{
+			AuthURL:                   "https://zoom.us/oauth/authorize",
+			TokenURL:                  "https://zoom.us/oauth/token",
+			ExplicitScopesRequired:    true,
```

Comment:
From the documentation, doesn't seem like scopes are required in the oauth request?

Reply:
@RajatPawar yes sorry I've wrongly enrolled. will update